src/kernels/jupyter/finder/remoteKernelFinder.ts

```diff
@@ -231,7 +230,7 @@ export class RemoteKernelFinder implements IRemoteKernelFinder, IExtensionSingle
      *
      * Remote kernel finder is resource agnostic.
      */
-    listContributedKernels(_resource: Resource): KernelConnectionMetadata[] {
+    listContributedKernels(_resource: Resource): RemoteKernelConnectionMetadata[] {
```

Comment:
The old type was incorrect, we cannot return just any type of kernel connection, we can only return remote kernel connections.
The new interface type fixes this.